configure.ml

```diff
@@ -728,6 +728,19 @@ let check_for_numlib () =
 let numlib =
   check_for_numlib ()
 
+(** OUnit, required for unit tests *)
+
+let check_for_ounit () =
+  let ounit,_ = tryrun camlexec.find ["query";"oUnit"] in
+  match ounit with
+  | ""  ->
+     die "OUnit not installed"
+  | _   ->
+     printf "You have OUnit installed. Good!\n"
+
+let ounit =
+  check_for_ounit ()
+
```

Comment:
Don't add this. We do not check test-suite dependencies. These are optional dependencies.

Reply:
OK.